public/form_validation.js

```diff
@@ -0,0 +1,43 @@
+let elements = document.querySelectorAll('p.required');
+let inputs = document.querySelectorAll('p.required>input');
+let selects = document.querySelectorAll('p.required>select');
+            
+for(let i=0; i<elements.length; i++){
+        let el = elements[i];
+        let input = el.querySelector('input');
+        if(input){
+        input.onblur = function() {        
+        if (input.value === "") {
```

Comment:
@Varannchik , I would add the extra check for the whitespaces "Validating the input
We want to know whether an input is empty. For our purpose, empty means:

The user hasn’t typed anything into the field
The user has typed one or more empty spaces, but not other characters"  https://www.freecodecamp.org/news/checking-if-an-input-is-empty-with-javascript-d41ed5a6195f/. Add trim function

Reply:
updated